_includes/related_posts.html

```diff
@@ -51,6 +58,13 @@
 					{% assign author = relevantPostAttributes[5] %}
 					{% assign postContent = relevantPostAttributes[6] %}
 					{% assign postDate = relevantPostAttributes[7] %}
+					{% assign abstract = relevantPostAttributes[8] %}
+					
+					{% if abstract %}
```

Comment:
@alvarogarcia7 Just to double check, relevantPostAttributes[8] doesn't error if there is nothing there? So all other posts without abstracts aren't affected.